rockusb/examples/rockusb.rs

```diff
@@ -311,7 +311,17 @@ fn list_available_devices() -> Result<()> {
     println!("Available rockchip devices");
     for d in devices.iter() {
         match d {
-            Ok(mut d) => println!("* {:?}", d.handle().device()),
+            Ok(mut d) => println!(
+                "* {:?} ({:?})",
+                d.handle().device(),
+                BootMode::from(
```

Comment:
Any reason for not using the boot_mode function here?

Reply:
I didn't want to construct a `Transport` object only to print a byte from the raw device handle. 
Alternatively, I could create a trait and implement it for `rusb::Device`, e.g:

```rust

trait RockusbExt {
    fn boot_mode(&self) -> Result<BootMode>;
}

let device = rusb::Device<GlobalContext>::new();
println!("bootmode {}", device.boot_mode());
```